python/tvm/relay/tensorrt.py

```diff
@@ -165,9 +216,17 @@ def add_whitelist_fn(attrs, args):  # pylint: disable=unused-variable
         ):
             print("add: bug in TRT with adding batched constants.")
             return False
+
+        # Skip this add op in TRT to avoid accuracy mismatch
+        if all([a == b for a, b in zip(args[0].checked_type.shape, [1, 546, 1, 1])]) and all(
```

Comment:
`list(map(int, args[x].checked_type.shape)) == [1, 546, 1, 1]` is slightly cleaner